pickle-processor/src/main/java/com/fourlastor/pickle/MethodsConverter.kt

```diff
@@ -51,11 +51,13 @@ class MethodsConverter(
                 propagate(e)
             }
             methodConverter.ignoredTest(methodName, name).also {
-                messager.warning("""
+                messager.warning(
+                    """
```

Comment:
We need to be sure that these multi-line strings are not broken because otherwise, CLI output becomes really ugly.

Reply:
Kotlin is not strict at all when it comes to multiline strings. I would recommend to revert these to be safe.

Reply:
2dee5b3 added tests to ensure the formatting is correct (generated them from `master`)